README.md

```diff
@@ -29,15 +57,36 @@ The Pieces Copilot Streamlit Bot is an interactive chatbot application built usi
    ```
    pip install -r requirements.txt
    ```
-
-4. Run the app
+3. Run the app
 
    ```python
    streamlit run pieces-bot.py
    ```
+## Additional Resources
+
+- [Pieces OS Python SDK](https://github.com/pieces-app/pieces-os-client-sdk-for-python)
+- [Pieces Documentation](https://docs.pieces.app/build/reference/python/)
+- [Streamlit](https://streamlit.io/)
+- [Websocket](https://websockets.readthedocs.io/en/stable/)
+
+## Contributing
+
+We welcome your contributions to Pieces! All changes (even with commit access) need a pull request & thumbs-up from a core dev
+
+We love ❤️ pull requests! Learn how to contribute in our guide: [contribution guide](https://github.com/pieces-app/example-typescript/blob/main/CONTRIBUTING.md).
+
+
+## Security
+
+If you discover any security-related issues, please email [security@pieces.app](mailto:security@pieces.app), or find a security contact in our [SECURITY.md](https://github.com/pieces-app/example-typescript/blob/main/SECURITY.md).
```

Comment:
This needs to be either removed from this repo or updated since there isn't a Security.md